packages/world_countries/CHANGELOG.md

```
@@ -1,3 +1,15 @@
+## 2.0.2
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)